builtin/nomad/jobspec/platform.go

```diff
@@ -2,23 +2,31 @@ package jobspec
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"strings"
+	"time"
 
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/nomad/structs"
```

Comment:
This package isn't a stable API. We import it into the Nomad Terraform provider and some of the plugins, and it's hugely painful and something we'd love to undo. Let's avoid pulling it into Waypoint and inflicting that pain on the Waypoint developers too 😀 

(That being said, although the alloc status consts are available in [`api/allocations.go`](https://github.com/hashicorp/nomad/blob/main/api/allocations.go#L17-L29) the eval and job status consts are not and absolutely should be. We'll need to fix this in the Nomad repo.)

Reply:
Updated!